Entities/Vehicles/Catapult/Catapult.as

```diff
@@ -13,6 +13,8 @@ const u8 charge_contrib = 35;
 const u8 cooldown_time = 45;
 const u8 startStone = 100;
 
+bool hadAttached = false;
```

Comment:
This should be a blob property, as this global variable will be shared across multiple catapults

Reply:
So would [this](https://github.com/transhumandesign/kag-base/blob/master/Entities/Items/Sponge/Sponge.as#L6) also be the same issue? This wasn't picked up when I improved sponges ages ago so I thought it was fine.